src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets

```diff
@@ -19,29 +19,29 @@ Copyright (c) .NET Foundation. All rights reserved.
                               $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '5.0'))">5.0</AnalysisLevel>
 
     <!-- EnableNETAnalyzers Allows analyzes to be disabled in bulk via msbuild if the user wants to -->
-    <EnableNETAnalyzers Condition="'$(EnableNETAnalyzers )' == '' And 
+    <EnableNETAnalyzers Condition="'$(EnableNETAnalyzers)' == '' And 
                                    '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And
                                    $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '5.0')) And
                                    $([MSBuild]::VersionGreaterThanOrEquals($(AnalysisLevel), '5.0'))">true</EnableNETAnalyzers>
-    <EnableNETAnalyzers Condition="'$(EnableNETAnalyzers )' == ''">false</EnableNETAnalyzers>
+    <EnableNETAnalyzers Condition="'$(EnableNETAnalyzers)' == ''">false</EnableNETAnalyzers>
     
     <!-- Compiler warning level, defaulted to 4. We promote it to 5 if the user has set analysis level to 5 or higher -->
     <WarningLevel Condition="'$(AnalysisLevel)' != '' And
                               $([MSBuild]::VersionGreaterThanOrEquals($(AnalysisLevel), '5.0'))">5</WarningLevel>
   </PropertyGroup>
-  <Import Project="..\analyzers\build\Microsoft.CodeAnalysis.NetAnalyzers.targets"
+  <Import Project="$(MSBuildThisFileDirectory)..\analyzers\build\Microsoft.CodeAnalysis.NetAnalyzers.targets"
           Condition="$(EnableNETAnalyzers)" />
   <ItemGroup Condition="$(EnableNETAnalyzers)">
     <Analyzer
       Condition="'$(Language)' == 'VB'"
-      Include="..\analyzers\Microsoft.CodeAnalysis.VisualBasic.NetAnalyzers.dll"
+      Include="$(MSBuildThisFileDirectory)..\analyzers\Microsoft.CodeAnalysis.VisualBasic.NetAnalyzers.dll"
       IsImplicitlyDefined="true" />
     <Analyzer
       Condition="'$(Language)' == 'C#'"
-      Include="..\analyzers\Microsoft.CodeAnalysis.CSharp.NetAnalyzers.dll"
+      Include="$(MSBuildThisFileDirectory)..\analyzers\Microsoft.CodeAnalysis.CSharp.NetAnalyzers.dll"
```

Comment:
It's always relative? I was thinking you could use something like GetDirectoryNameOfFileAbove, but that also assumes there'd be a well known file that's unlikely to change or move.